### PR TITLE
Address Telegram HTTP 500s when receiving message from Telegram in discussion group

### PR DESCRIPTION
**Pull request by joeyorlando**
# Which issue(s) this PR fixes

Closes https://github.com/grafana/oncall/issues/3621

## Checklist

- [x] Unit, integration, and e2e (if applicable) tests updated
- [ ] Documentation added (or `pr:no public docs` PR label added if not required)
- [x] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

### CODE REVIEW

File: engine/apps/telegram/updates/update_manager.py
Comment (joeyorlando) [inline engine/apps/telegram/updates/update_manager.py:1]: another possibility is to add an "update handler" for messages originating directly in a channel (not a discussion group). For these HTTP 500s that we are seeing, they occur when a message is sent from the official Telegram user directly in a channel, not the channel's discussion group.

Right now we're handling these updates w/ the `apps.telegram.updates.update_handlers.channel_to_group_forward.ChannelToGroupForwardHandler` handler (this handler is determined [here](https://github.com/grafana/oncall/pull/3622/files#diff-440894128e379fb8abd0627e0d193d04dbbd8a26512ef0abcffa76bbf9ee6a46R27-R42)).

For now, what I've done in this PR seems like a simpler approach to arrive at the same solution, but open to thoughts/comments.

File: engine/conftest.py [hunk 1]
```diff
@@ -443,19 +442,6 @@ def _make_slack_message(alert_group=None, organization=None, **kwargs):
...
-        """The client with logged in user"""
-
-        client = APIClient()
-        client.force_login(user)
-
-        return client
-
-    return _client_with_user
-
-
```
Comment (joeyorlando) [inline engine/conftest.py:458]: this doesn't appear to be used anywhere

File: engine/apps/telegram/tests/test_webhook.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
+from unittest.mock import patch
+
+import pytest
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIClient
+
+url = reverse("telegram:incoming_webhook")
```
Comment (matiasb) [inline engine/apps/telegram/tests/test_webhook.py:8]: nit, I guess I would have this as a local var in each test initial setup

**Review by matiasb — APPROVED** [review]
LGTM

**Pull request merged**